setup.cfg

```diff
@@ -1,3 +1,34 @@
+[metadata]
+name = pymms
```

Comment:
What if we called it just mms? If this is going to be part of pypi then the py- is implied, it also looks like py-mms already exists: https://pypi.org/project/py-mms/

Reply:
That is what the package import is named, and it should be the same as this is the name it will be installed as in the local site-packages. If we decide to push it to PyPi, we should probably name it something more descriptive. Also, plenty of packages on PyPi prefixed with `py` to match their import name.